packages/rustworkx/meta.yaml

```diff
@@ -1,15 +1,17 @@
 package:
   name: rustworkx
-  version: 0.17.0a3
+  version: 0.17.1
   top-level:
     - rustworkx
   tag:
     - rust
 source:
-  url: https://github.com/IvanIsCoding/rustworkx/releases/download/v0.17.0a3/rustworkx-0.17.0a3.tar.gz
-  sha256: 8bd0c295134e2b0c03808d4e69428b41153849db6488839084a78793c337f191
+  url: https://github.com/Qiskit/rustworkx/archive/refs/tags/0.17.1.tar.gz
+  sha256: dc248da9cc364b81ac67f5d67b626ee3f03a1f6299f7a9b4d0a0501548715365
 build:
-  script: export RUSTFLAGS="$RUSTFLAGS -C target-feature=+atomics,+bulk-memory,+mutable-globals"
+  script: |
+    rustup component add rust-src
```

Comment:
@hoodmane WDYT about this? I am not sure if it is a good idea for individual package to run this command, as it will make global effect (AFAIK).

Reply:
@hoodmane Do you think that `rustup component add rust-src` can overwrite the custom stdlib that we built with `wasm-exception` enabled?

Rust build toolchain is always confusing to me...